demo/src/css/styles.css

```diff
@@ -6,3 +6,17 @@ input[type="number"]::-webkit-outer-spin-button {
     -webkit-appearance: none;
     margin: 0;
 }
+input:not([type="checkbox"], [type="radio"]),
```

Comment:
In general because we're already using a standard UI library, you should avoid custom CSS wherever possible. I don't think any of these rules are necessary (especially not the font one, see my other comment about Google Fonts), but you can provide screenshots here to show the differences.